test-integration/samples/vue-ws-theme/.yo-rc.json

```diff
@@ -26,7 +26,7 @@
     "otherModules": [],
     "packageFolder": "tech/jhipster/sample",
     "packageName": "tech.jhipster.sample",
-    "prodDatabaseType": "postgresql",
+    "prodDatabaseType": "mariadb",
```

Comment:
maybe nice to rename the test name if it's the only configuration with this database

Reply:
It’s the only test with the combination h2 + mariadb + dev.